test/models/group_test.rb

```diff
@@ -7,6 +7,13 @@ class GroupTest < ActiveSupport::TestCase
 
   test('belongs to #author') { assert_kind_of(User, @subject.author) }
 
+  test('belongs to #project') { assert_kind_of(Project, @subject.project) }
+
+  test('can exist without a #project') do
```

Comment:
At this stage let's assume the groups cannot exist without a project, so we want the opposite of this functionality.